resources/function_help/json/to_dm

```diff
@@ -7,10 +7,11 @@
     {"arg":"coordinate","description":"A latitude or longitude value."},
     {"arg":"axis","description":"The axis of the coordinate. Either 'x' or 'y'."},
     {"arg":"precision", "description":"Number of decimals."},
-    {"arg":"formatting", "optional": true, "default":"", "description":"Designates the formatting type. Acceptable values are NULL, 'aligned' or 'suffix'."}
+    {"arg":"formatting", "optional": true, "default":"", "description":"Designates the formatting type. Acceptable value is NULL (default), 'aligned' or 'suffix'."}
```

Comment:
This reads worse to me -- why the change?

Reply:
- the `"default" : ""` does not render in QGIS, hence the precision next to the NULL value
- with a plural form, I'd expect an "AND" list so either `Acceptable values are NULL, 'aligned' and 'suffix'` or use the singular form with "OR".

Reply:
Mmm... it doesn't read well to me -- it's just awkward grammar. I'm not an English teacher, but I'm 99.9% sure the original form is correct.

Reply:
Reverted...